django/common/scripts/merging/merge_concat_without_separator.py

```diff
@@ -1,5 +1,5 @@
-def concat_without_separator(*args):
+def merge_concat_without_separator(*args):
     """Merging script with a simple concatenation, no separator"""
     values = [str(v) for v in args if v is not None]
     separator = ""
-    return separator.join(values)
+    return separator.join(values)
```

Comment:
I'm not sure this blank line should have been removed. Have you installed the pre-commit hooks?